### ISSUE TITLE
ansible_post_tasks: comma-separated value of paths results in ansible failure

### ISSUE THREAD

**Issue by gregwhorley** [splunk/splunk-ansible#587]
According to your docs, the value of `ansible_post_tasks` is a string of comma-separated paths or URLs to custom ansible playbooks: https://github.com/splunk/splunk-ansible/blob/5b1c31409cf64cd31c45f065557bb18ba0a49d94/docs/advanced/default.yml.spec.md#spec

If I set more than one `file://` path as the value of that option:
```
ansible_post_tasks: file:///tmp/defaults/remove_outputs_conf_playbook.yml,file:///tmp/defaults/enterprise_license_copy_playbook.yml
```

The ansible task fails:
```
TASK [Fetch adhoc playbooks] ***************************************************
fatal: [localhost]: FAILED! => {
    "changed": false,
    "dest": "/opt/container_artifact/enterprise_license_copy_playbook.yml",
    "elapsed": 0,
    "url": "file:///tmp/defaults/remove_outputs_conf_playbook.yml,file:///tmp/defaults/enterprise_license_copy_playbook.yml"
}

MSG:

Request failed: <urlopen error [Errno 2] No such file or directory: '/tmp/defaults/remove_outputs_conf_playbook.yml,file:///tmp/defaults/enterprise_license_copy_playbook.yml'>
```

The ansible run output shows that the last path in the comma-separated list is found and is executed:

```
Request failed: <urlopen error [Errno 2] No such file or directory: '/tmp/defaults/remove_outputs_conf_playbook.yml,file:///tmp/defaults/enterprise_license_copy_playbook.yml'>
...ignoring
Thursday 31 December 2020  21:24:27 +0000 (0:00:00.309)       0:00:32.209 *****
included: /tmp/defaults/enterprise_license_copy_playbook.yml for localhost
Thursday 31 December 2020  21:24:27 +0000 (0:00:00.035)       0:00:32.244 *****

TASK [Make enterprise directory under etc/licenses] ****************************
ok: [localhost]
Thursday 31 December 2020  21:24:27 +0000 (0:00:00.106)       0:00:32.350 *****

TASK [Copy license from var/secrets] *******************************************
ok: [localhost]
Thursday 31 December 2020  21:24:28 +0000 (0:00:00.624)       0:00:32.975 *****
included: /opt/ansible/roles/splunk_common/tasks/check_for_required_restarts.yml for localhost
Thursday 31 December 2020  21:24:28 +0000 (0:00:00.036)       0:00:33.012 *****
```

**Comment by nwang92** [comment]
It looks like this functionality may have changed such that the docs are wrong. Let me see if I can restore this feature so that a comma-separated string works!

**Comment by nwang92** [comment]
This will be available in newer Splunk image versions > 8.1.1. Alternatively, you should also be able to pick up this fix/change in the nightly `splunk/splunk:edge`.
